sdk/identity/Azure.Identity/tests/VisualStudioCodeCredentialLiveTests.cs

```diff
@@ -82,7 +82,7 @@ public async Task AuthenticateWithVscCredential_EmptySettingsFile()
         }
 
         [Test]
-        [RunOnlyOnPlatforms(Windows = true, OSX = true)] // Comment this attribute to run this tests on Linux with Libsecret enabled
+        [RunOnlyOnPlatforms(Windows = true, OSX = true, ContainerNames = new[] { "ubuntu_netcore2_keyring" })]
```

Comment:
@schaabs we finally have tests that run on linux with enabled keyring. Maybe other identity teams would need that too.